StyleGuide.md

```diff
@@ -1070,7 +1070,32 @@ CREATE DATABASE movr PRIMARY REGION "gcp-us-east1" REGIONS "gcp-us-east1", "gcp-
 ```
 
 For more information about the `remote_include` tag, see the README in the [jekyll-remote-include](https://github.com/cockroachdb/jekyll-remote-include) repo.
+  
+#### Filter tabs
+  
+On some pages in our docs, there are tabs at the top of the page that will link to different pages at different hyperlinks. For example, in the [Install CockroachDB docs](https://www.cockroachlabs.com/docs/stable/install-cockroachdb.html), there are links to the Mac, Linux, and Windows pages at the top of the page.
+  
+Use [`filter-tabs.md`](https://github.com/cockroachdb/docs/blob/master/_includes/filter-tabs.md) to specify these tabs for any `cockroachcloud` docs or docs for CockroachDB v21.2 and later.
+
+**Note:** this include file only produces tabs that link to different URLs/pages. It cannot be used for creating tabs within a single page.
+
+The general process to follow and use this is as follows:
+  
+1. Identify each page to be linked to a filter tab.
+    - Make a note of each HTML page name (e.g., `install-cockroachdb-mac.html`).
+    - Identify the desired HTML-supported tab name (e.g., `Install on <strong>Mac</strong>`).
+2. Create an include Markdown file within `_includes/<CRDB version>/filter-tabs` with the following structure:
+    ```
+    {% assign tab_names_html = "Tab Name 1;Tab Name 2;Tab Name 3" %}
+    {% assign html_page_names = "page-name-1.html;page-name-2.html;page-name-3.html" %}
```

Comment:
For a future improvement, we could rename `html_page_names` to `html_page_filenames`, for clarity.

Reply:
https://cockroachlabs.atlassian.net/browse/DOC-5506